src/pages/home/sidebar/SidebarLinksData.tsx

```diff
@@ -252,14 +204,17 @@ export default compose(
         },
         isLoadingApp: {
             key: ONYXKEYS.IS_LOADING_APP,
+            selector: (s) => s,
```

Comment:
this is not needed , remove compose usage, instead of `withNetwork` use `useNetwork` , instead `withNAvigationFocus` use `useIsFocused` and properly type `withOnyx` then you can get rid of selectors